lrauv_ignition_plugins/src/ScienceSensorsSystem.cc

```diff
@@ -582,16 +651,38 @@ void ScienceSensorsSystem::PostUpdate(const ignition::gazebo::UpdateInfo &_info,
       {
         // Increment for next point in time
         this->dataPtr->timeIdx++;
+
+        // Publish science data at the next timestamp
         this->dataPtr->PublishData();
       }
     }
 
+    // Publish every n iters so that VisualizePointCloud plugin gets it.
+    // Otherwise the initial publication in Configure() is not enough.
+    if (this->dataPtr->repeatPubTimes % 10000 == 0)
```

Comment:
We should revisit this, the service call that `VisualizePointCloud` makes at startup is meant to address the case when it misses the initial publish from this system, or when that plugin is simply loaded later.